pages/tools/geode_objects.js

```diff
@@ -0,0 +1,64 @@
+const geode_objects = {
+    BRep: {
+        tooltip: 'BRep',
+        image: 'BRep.svg',
+    },
+    EdgedCurve2D: {
+        tooltip: 'EdgedCurve2D',
+        image: 'edged_curve2d.svg',
+    },
+    EdgedCurve3D: {
+        tooltip: 'EdgedCurve3D',
+        image: 'edged_curve3d.svg',
+    },
+    Graph: {
+        tooltip: 'Graph',
+        image: 'edged_curve.svg',
+    },
+    // HybridSolidy3D: {
```

Comment:
Solidy?

Reply:
I can add HybridSolidy3D but i need the svg file that goes along with it, it wasn't provided if I remember well